src/assignment/queries/get-assignment-by-uuid.graphql

```diff
@@ -65,7 +66,21 @@ query getAssignmentByUuid($id: uuid!) {
 			assignment_id
 		}
 		contributors {
+			profile {
+				avatar
+				user_id
```

Comment:
id or profile id, not user id

each user account has a user entry and a profile entry
most of the other tables use the profile id and not the user id to link to a user account